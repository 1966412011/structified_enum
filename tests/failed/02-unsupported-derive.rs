use structified_enum::structify;

#[structify]
#[derive(Shit)]
enum Foo {}

fn main() {}
