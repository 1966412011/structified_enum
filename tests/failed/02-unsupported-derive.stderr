error: unsupported derive. It only supports `Clone`, `Copy`, `PartialEq`, `Eq`, `PartialOrd`, `Ord`, `Hash`, `Default`, `Debug`.
 --> tests/failed/02-unsupported-derive.rs:4:10
  |
4 | #[derive(Shit)]
  |          ^^^^
